components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -2505,63 +2505,65 @@ public static boolean isAnalyticsEnabled() {
         return propertyMap;
     }
 
+	public static void addDefinedAllSequencesToRegistry(UserRegistry registry,
```

Comment:
Formatting issue of this method is already fixed while merging 1.10.x branch to master branch